lib/ovirt/vm.rb

```diff
@@ -332,5 +352,19 @@ def create_template(options={})
       raise TemplateAlreadyExists, err.message if err.message.include?("Template name already exists")
       raise
     end
+
+    private
+
+    def converted_cloud_init(content)
+      ovirt_cloud_init_keys = %w(active_directory_ou authorized_ssh_keys dns_search dns_servers domain host_name input_locale nic_configurations org_name regenerate_ssh_keys root_password system_locale timezone ui_language user_locale user_name)
+
+      require 'yaml'
+      raw_content = YAML.load(content)
+
+      hash = ovirt_cloud_init_keys.each_with_object({}) { |k, h| h[k] = raw_content.delete(k) }
```

Comment:
Actually..wait...why do you have to remove the keys from raw_content?  You don't even return it.